### DISCUSSION TITLE
How about a new minor version to help sqlx?

### DISCUSSION THREAD

**Discussion by nappa85** [SeaQL/sea-orm#498] [Ideas]
Looking at https://github.com/launchbadge/sqlx/pull/1505 I think it would be cool if you bump a new minor version, since sqlx guys are worried they could broke something releasing 0.6 this quick, but it's be a really important upgrade and I think it needs to be done quickly.

What do you think?

**Comment by billy1624** [comment]
Hey @nappa85, sorry for the delay. Yeah, we should! I can open a PR to test depending on it later this week. It has been busy loll

**Comment by nappa85** [comment]
I don't know if 0.6.0 served the purpose, but it doesn't make sense I act as an intermediary, you can check with sqlx guys directly

**Reply by billy1624** [reply]
> You can check with sqlx guys directly

Sure! Thxx

**Comment by nappa85** [comment]
Hello guys,
same thread, can you check this sqlx issue https://github.com/launchbadge/sqlx/issues/1796 and give your opinion?
I think sea-orm is involved

**Reply by tyt2y3** [reply]
Oh I just read the whole thread and honestly I lean more onto "frequent releases" (as we do now) but I could not think of a better solution for the entire Rust ecosystem either. May be I will have some new thoughts tomorrow.

**Reply by paolobarbolini** [reply]
> Oh I just read the whole thread and honestly I lean more onto "frequent releases" (as we do now) but I could not think of a better solution for the entire Rust ecosystem either. May be I will have some new thoughts tomorrow.

Would this work for sea-orm?

https://github.com/launchbadge/sqlx/issues/1796#issuecomment-1098879753

**Reply by tyt2y3** [reply]
My two cents https://github.com/launchbadge/sqlx/issues/1796#issuecomment-1101241228
